interface-definitions/service_dns_forwarding.xml.in

```diff
@@ -793,6 +793,184 @@
+                      <leafNode name="timeout">
+                        <properties>
+                          <help>Zone retrieval timeout</help>
+                          <valueHelp>
+                            <format>u32:1-3600</format>
+                            <description>Request timeout in seconds</description>
+                          </valueHelp>
+                          <constraint>
+                            <validator name="numeric" argument="--range 1-3600"/>
+                          </constraint>
+                        </properties>
+                        <defaultValue>20</defaultValue>
+                      </leafNode>
+                      <node name="refresh">
+                        <properties>
+                          <help>Zone caching options</help>
+                        </properties>
+                        <children>
+                          <leafNode name="on-reload">
+                            <properties>
+                              <help>Retrieval zone only at startup and on reload</help>
+                              <valueless/>
+                            </properties>
+                          </leafNode>
+                          <leafNode name="interval">
+                            <properties>
+                              <help>Periodic zone retrieval interval</help>
+                              <valueHelp>
+                                <format>&lt;minutes&gt;</format>
+                                <description>Retrieval interval in minutes</description>
+                              </valueHelp>
+                              <valueHelp>
+                                <format>&lt;minutes&gt;m</format>
+                                <description>Retrieval interval in minutes</description>
+                              </valueHelp>
+                              <valueHelp>
+                                <format>&lt;hours&gt;h</format>
+                                <description>Retrieval interval in hours</description>
+                              </valueHelp>
+                              <valueHelp>
+                                <format>&lt;days&gt;d</format>
+                                <description>Retrieval interval in days</description>
+                              </valueHelp>
+                              <constraint>
+                                <regex>[1-9]([0-9]*)([mhd]{0,1})</regex>
+                              </constraint>
+                            </properties>
+                            <defaultValue>24h</defaultValue>
+                          </leafNode>
+                        </children>
+                      </node>
+                      <leafNode name="retry-interval">
+                        <properties>
+                          <help>Retry interval after zone retrieval errors</help>
+                          <valueHelp>
+                            <format>u32:1-2147483647</format>
```

Comment:
Retry every 24855 days? I think the bounds are to relaxed. I'd go for 1 - 86400 so from one second to once a day.

Reply:
fixed